sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/workspace_asset_reference.py

```diff
@@ -19,6 +19,7 @@
 class WorkspaceAssetReference(Asset):
     """Workspace Model Reference.
 
+    This is for SDK internal use only, might be deprecated in the future.
```

Comment:
If we really want to stop people, we could go further here and actually add a deprecation warning if customers use this. But given that it's highly unlikely customers stumbled upon this, that might be overkill...

Reply:
Let's keep this for the time being!